docs/styles.rst

```diff
@@ -0,0 +1,137 @@
+Styles
+======
+
+VSG supports several predefined styles.
+They can be used with the **--style** command line option.
+
+The table below lists the built in styles available
+
++---------------+--------------------------------------------+
+| Style         | Description                                |
++---------------+--------------------------------------------+
+| base          | Current rule set                           |
+| disabled      | All rules are disabled                     |
```

Comment:
I would remove `disabled` style. It is super easy to disable all rules in the yaml config file, so user can do it.

Reply:
I suppose you could argue that `disabled` is not really a style.  I can remove it.

Reply:
Update has been pushed.